build.gradle

```diff
@@ -2,11 +2,11 @@ import com.github.jk1.license.filter.LicenseBundleNormalizer
 import com.github.jk1.license.render.JsonReportRenderer
 
 plugins {
-    id 'com.palantir.docker' version '0.25.0' apply false
-    id 'org.owasp.dependencycheck' version '8.3.1'
-    id 'com.gorylenko.gradle-git-properties' version '2.4.1' apply false
-    id 'com.github.jk1.dependency-license-report' version '2.5' apply false
-    id 'de.undercouch.download' version '5.4.0' apply false
+    id 'com.palantir.docker' version '0.36.0' apply false
+    id 'org.owasp.dependencycheck' version '10.0.4'
+    id 'com.gorylenko.gradle-git-properties' version '2.4.2' apply false
+    id 'com.github.jk1.dependency-license-report' version '2.9' apply false
+    id 'de.undercouch.download' version '5.6.0' apply false
```

Comment:
suggestion: While we are here it is worth migrating to th2 gradle plugin - it should simplify our life in future